### DISCUSSION TITLE
Tracking Korean Localization Progress

### DISCUSSION THREAD

**Discussion by seokho-son** [cncf/glossary#1116] [Show and tell]
Discussion to manage and track Korean localization progress.

## Information: Korean localization
 - #288 
   - Currently, there are 3 Korean contents approvers
   - We welcome contributors. 기여를 환영합니다.
 - We use a development branch (https://github.com/cncf/glossary/tree/dev-ko). 
   - Contributors need to use `dev-ko` as the base branch, and the target branch of a PR should be `dev-ko`
 - For more details and any information that you need, please don't hesitate and ask on the CNCF slack channel: [#glossary-localization-korean](https://cloud-native.slack.com/archives/C02PC0MLQKU).

## Contribution procedure
1. Comment below the term that you would like to localize, focus on one term per PR.
   - Check that no one else has claimed the term in the comments below. 
   - Check the latest English document in main branch. 
     - If the document is removed or `status: Deprecated`, we don't recommend to work on that term.
     - A document with `status: Feedback Appreciated` has a low-priority.
   - A Korean approver will assign a contributor to the task to localize a term. (It is to prevent duplicate works)
2. Copy a file (`content/en/xx.md`) for original English document to `content/ko/` directory. (need to use the same file name with English)
3. Localize (translate) the English contents inside.
    - You don't need to translate `status: Completed` parameter in the document.
4. Open a PR (ex: #334)
    - Contributors need to use `dev-ko` as the base branch, and the target branch of a PR should be `dev-ko`
    - Please use concise and informative English PR title. (ex: `Localize cloud computing for Korean`)
      - 내용 및 코멘트는 한글로 작성하셔도 좋습니다.
5. Participated in review process
    - Korean approvers will review the PR.
    - Update the PR according to reviewers' comments (or discuss before update)

## Priorities (already achieved ! by PR #801)
We need at least 10 terms in Korean before go to live (v1), below are some suggested terms to work (but not limited):

|Num| File to localize | Assignee | Ref (#PR) | Note |
|---|--------|----------|---------------|------|
|conf|[content/en/_index.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/_index.md)|@seokho-son|#291|      |
|conf|[content/en/contribute/_index.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/contribute/_index.md)|@seokho-son|#291|update @jihoon-seo|
|conf|[content/en/contributor-ladder/_index.md](https://github.com/cncf/glossary/blob/main/content/en/contributor-ladder/_index.md)|@seokho-son|#770| |
|conf|[content/en/style-guide/_index.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/style-guide/_index.md)|@seokho-son|#291|update @jihoon-seo|
|01|[content/en/cloud_native_tech.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/cloud_native_tech.md)|@seokho-son|#291|      |
|02|[content/en/cloud_native_apps.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/cloud_native_apps.md)|@jihoon-seo|#743|      |
|03|[content/en/cloud_computing.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/cloud_computing.md)|@seokho-son|#334|      |
|04|[content/en/microservices.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/microservices.md)|@yunkon-kim|#528|      |
|05|[content/en/monolithic_apps.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/monolithic_apps.md)|@yunkon-kim|#504|      |
|06|[content/en/container.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/container.md)|@seokho-son|#382|      |
|07|[content/en/virtual_machine.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/virtual_machine.md)|@yunkon-kim|#424|      |
|08|[content/en/containerization.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/containerization.md)|@jihoon-seo|#742|      |
|09|[content/en/service_mesh.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/service_mesh.md)|@jihoon-seo|#745|      |
|10|[content/en/devops.md](https://github.com/cncf/glossary/blob/dev-ko/content/en/devops.md)|@jihoon-seo|#726|      |
|...|Not limited| | |      |

## l10n task list 

Please check a button where you wish to contribute to. 
(checked button means WIP or Completed)

 - [x] _index.md
 - [x] _TEMPLATE.md
 - [x] abstraction.md
 - [x] agile-software-development.md
 - [x] api-gateway.md
 - [x] application-programming-interface.md
 - [x] auto-scaling.md
 - [x] bare-metal-machine.md
 - [ ] blue-green-deployment.md
 - [x] canary-deployment.md
 - [ ] chaos-engineering.md
 - [ ] client-server-architecture.md
 - [x] cloud-computing.md
 - [x] cloud-native-apps.md
 - [ ] cloud-native-security.md
 - [x] cloud-native-tech.md
 - [x] cluster.md
 - [x] container.md
 - [ ] container-image.md
 - [x] containerization.md
 - [x] containers-as-a-service.md
 - [x] continuous-delivery.md
 - [ ] continuous-deployment.md
 - [x] continuous-integration.md
 - [ ] database-as-a-service.md
 - [ ] data-center.md
 - [x] debugging.md
 - [x] devops.md
 - [ ] devsecops.md
 - [ ] digital-certificate.md
 - [x] distributed-apps.md
 - [x] distributed-systems.md
 - [ ] event-driven-architecture.md
 - [ ] firewall.md
 - [ ] function-as-a-service.md
 - [ ] gitops.md
 - [x] horizontal-scaling.md
 - [x] idempotence.md
 - [ ] immutable-infrastructure.md
 - [ ] infrastructure-as-a-service.md
 - [x] infrastructure-as-code.md
 - [x] kubernetes.md
 - [ ] load-balancer.md
 - [ ] loosely-coupled-architecture.md
 - [x] managed-services.md
 - [x] microservices.md
 - [x] monolithic-apps.md
 - [ ] mutual-transport-layer-security.md
 - [ ] nodes.md
 - [ ] observability.md
 - [ ] platform-as-a-service.md
 - [ ] policy-as-code.md
 - [x] portability.md
 - [ ] reliability.md
 - [x] scalability.md
 - [ ] search.md
 - [ ] security-chaos-engineering.md
 - [ ] self-healing.md
 - [x] serverless.md
 - [x] service.md
 - [x] service-discovery.md
 - [x] service-mesh.md
 - [ ] service-proxy.md
 - [ ] shift-left.md
 - [x] site-reliability-engineering.md
 - [ ] software-as-a-service.md
 - [ ] stateful-apps.md
 - [ ] stateless-apps.md
 - [ ] tightly-coupled-architectures.md
 - [ ] transport-layer-security.md
 - [x] version-control.md
 - [ ] vertical-scaling.md
 - [ ] virtualization.md
 - [x] virtual-machine.md
 - [ ] zero-trust-architecture.md

**Comment by seokho-son** [comment]
06 | content/en/container.md
-- | --

제가 진행하겠습니다!

**Comment by yunkon-kim** [comment]
| 07 | content/en/virtual_machine.md |
|:---:|:-----------------------------------:|

진행해보겠습니다 😃

**Comment by seokho-son** [comment]
@yunkon-kim 
#382 
를 예시로 생각하시면 될 것 같습니다. :) 혹시 궁금한 사항이 있으시면 알려주세요~~

**Comment by yunkon-kim** [comment]
| 05 | content/en/monolithic_apps.md |
|:---:|:------------------------------------:|

다음으로 monolithic_apps를 진행하겠습니다 😃

**Comment by yunkon-kim** [comment]
| 04 | content/en/microservices.md |
|:---:|:---------------------------------:|

다음으로 microservices를 진행하겠습니다

**Comment by jihoon-seo** [comment]
10 | content/en/devops.md
-- | --

제가 DevOps 하겠습니다~~

**Comment by jihoon-seo** [comment]
08 | content/en/containerization.md
-- | --

Containerization 하겠습니다~~

**Comment by jihoon-seo** [comment]
02 | content/en/cloud_native_apps.md
-- | --

Cloud Native Apps 하겠습니다~~

**Comment by jihoon-seo** [comment]
09 | content/en/service_mesh.md
-- | --

Service Mesh 하겠습니다~~

**Comment by jihoon-seo** [comment]
conf | content/en/contribute/_index.md
-- | --

제가 하겠습니다~~

**Comment by jihoon-seo** [comment]
conf | content/en/style-guide/_index.md
-- | --

제가 하겠습니다~~

**Comment by seokho-son** [comment]
https://github.com/cncf/glossary/blob/main/content/en/contributor-ladder/_index.md 도 추가하면 좋을 것 같습니다. :)

**Comment by hongju-jeong** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/service.md

제가 진행하겠습니다!

**Reply by seokho-son** [reply]
Merged! Thanks!

**Comment by SW0000J** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/debugging.md

진행해보겠습니다!

**Comment by SeolRoh** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/service-discovery.md
제가 하겠습니다~

**Comment by kidae92** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/cluster.md

진행해보겠습니다.

**Comment by Chiwon-Park** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/distributed-systems.md

진행하겠습니다.

**Comment by yuri0329** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/infrastructure-as-code.md  
진행하겠습니다.

**Comment by yunkon-kim** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/auto-scaling.md
진행하겠습니다.

Permalink(dev-ko): https://github.com/cncf/glossary/blob/f9ff459a648ac43760ac69a9be9d1a965ba46a82/content/en/auto-scaling.md

**Reply by yunkon-kim** [reply]
I want to note the permalink to indicate the original English content of the localized content. ^^

Note - a test if the permalink persists when `dev-ko` branch changes
: Assume that `dev-ko` is deleted. The message below showed up.
![image](https://user-images.githubusercontent.com/7975459/185009621-317432f8-c02e-4cdc-a388-1c69a36115aa.png)

**Comment by hongju-jeong** [comment]
service Localze PR [#1123](https://github.com/cncf/glossary/pull/1123#discussion_r935120278) 논의에서 architecture를 구조가 아닌 아키텍처로 번역하기로 결정했습니다. 한글화된 모든 architecture 부분을 아키텍처로 수정하겠습니다.

**Comment by hongju-jeong** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/serverless.md
진행하겠습니다.

**Comment by bconfiden2** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/scalability.md
진행하겠습니다!

**Comment by jinnypark9393** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/kubernetes.md
진행해보겠습니다 😄

**Comment by bconfiden2** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/distributed-apps.md
진행하겠습니다!

**Comment by Chiwon-Park** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/portability.md
진행하겠습니다!

**Comment by SeolRoh** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/idempotence.md
진행하겠습니다!

**Comment by yuri0329** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/agile-software-development.md
진행하겠습니다!

**Comment by seokho-son** [comment]
https://github.com/cncf/glossary/blob/main/content/en/abstraction.md 
한글화 진행하겠습니다. :)

**Comment by kidae92** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/containers-as-a-service.md
한글화 진행하겠습니다 !

**Comment by SeolRoh** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/version-control.md
한글화 진행하겠습니다:)

**Comment by yuri0329** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/managed-services.md
한글화 진행하겠습니다 :)

**Comment by Virusuki** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/continuous-delivery.md
한글화 진행하겠습니다! :)

**Comment by bconfiden2** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/site-reliability-engineering.md
한글화 진행하겠습니다 !

**Comment by bconfiden2** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/horizontal-scaling.md
한글화 진행하겠습니다 !!

**Comment by jongwooo** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/continuous-integration.md
한글화 진행하겠습니다:)

**Reply by yunkon-kim** [reply]
@jongwooo 안녕하세요. 반갑습니다!!

원활한 소통을 위해 한글화팀 슬랙 채널([#glossary-localization-korean](https://cloud-native.slack.com/archives/C02PC0MLQKU))에 참여해 주시기 바랍니다 ^^

**Reply by jongwooo** [reply]
@yunkon-kim 알려주셔서 감사합니다:)

**Comment by yunkon-kim** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/api-gateway.md
한글화 진행하겠습니다. :-)

**Comment by yunkon-kim** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/application-programming-interface.md
한글화 진행하겠습니다 ~ ^^

**Comment by yunkon-kim** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/bare-metal-machine.md
↑↑↑ 한글화 진행하겠습니다 ~

**Comment by jo1132** [comment]
~~Localize: https://github.com/cncf/glossary/blob/main/content/en/load-balancer.md~~
~~한글화 진행하겠습니다~~
아직 개발이 완료된 문서가 아니기 때문에, 다른 문서 한글화 진행하겠습니다!

**Reply by seokho-son** [reply]
@jo1132 감사합니다! :)
참고로 해당 영문 문서는 아직 개발이 완료된 상태가 아니라 (상태: feedback appreciated) 사이트상에서는 아직 보이지 않는 문서입니다. 나중에 변경되거나 수정될 수 있습니다. 완료된 문서에 대한 기여를 하시는 것이 더 좋을 것 같기도 하네요. :) 자유롭게 선택하시면 됩니다!

**Reply by jo1132** [reply]
@seokho-son  빠른 답변 감사합니다! 
그렇다면 개발이 완료된 문서를 다시 찾아보겠습니다. 감사합니다!

**Comment by jo1132** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/canary-deployment.md
한글화 진행하겠습니다!!

**Comment by dlwmznzl1** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/self-healing.md
한글화 진행하겠습니다!

**Reply by yunkon-kim** [reply]
@dlwmznzl1 안녕하세요!

한글화팀 슬랙 채널에 참여하셨을까요? 아직이시라면 참여를 바랍니다 ^^

**Reply by dlwmznzl1** [reply]
넵 슬랙 가입했습니다! 채널 참여 이름 깃 ID인 dlwmznzl1로 변경했습니다.

**Comment by yuri0329** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/reliability.md
한글화 진행하겠습니다 !

**Comment by SeolRoh** [comment]
Localize : https://github.com/cncf/glossary/blob/main/content/en/idempotence.md
멱등성 업데이트 되어 한글화 진행하겠습니다!

**Comment by ethankim417** [comment]
Localize: https://github.com/cncf/glossary/blob/main/content/en/blue-green-deployment.md
한글화 진행하겠습니다!

**Comment by 2yong2tae** [comment]
Localize: content/en/digital-certificate.md

digital-certificate.md 한글화를 진행하고 싶습니다.
첫 기여라 가이드에 맞춰 진행하겠습니다. 감사합니다!
